KubeArmor/feeder/feeder.go

```diff
@@ -621,7 +621,10 @@ func (fd *Feeder) PushLog(log tp.Log) {
 		if len(log.Action) > 0 {
 			pbAlert.Action = log.Action
 		}
-
+		pbAlert.ProcessName = log.ProcessName
+		pbAlert.ProcessPath = log.ProcessPath
+		pbAlert.ParentProcessName = log.ParentProcessName
```

Comment:
Given the inconsistency that we have for getting process names ... we can simply abandon it .. We do not need `ProcessName` and `ParentProcessName` if we can't get the values consistently.

Note: We are not able to get `ProcessName` correctly in case the process execution is denied because of kubearmor policy. If the process execution is denied then the LSM hooks blocks forking itself and the point at which it is blocked, the process name may not have been copied into appropriate task context for kubearmor to use. Thus in this case it keeps showing parent process name and there does not seem to be an easy way to fix this.